openshift/celery-worker.yaml

```diff
@@ -305,6 +305,24 @@ objects:
                   name: ${NAME}-logging
                   key: django-logging-file
                   optional: true
+            - name: S3_BUCKET_NAME
+              valueFrom:
+                configMapKeyRef:
+                  name: ${NAME}-app
+                  key: s3-bucket-name
+                  optional: false
+            - name: S3_STORAGE_PREFIX
+              valueFrom:
+                configMapKeyRef:
+                  name: ${NAME}-app
+                  key: s3-bucket-path
```

Comment:
Why the rename? We should be consistent between the name and key to make future searching/refactoring efforts easier. I have no strong preference for `S3_STORAGE_PREFIX` versus `S3_BUCKET_PATH`, and if the latter looks better, let's make them all like that.

Reply:
An oversight. It is now `S3_BUCKET_PATH` everywhere.